auction-server/src/api.rs

```diff
@@ -418,7 +418,7 @@ pub async fn start_api(run_options: RunOptions, store: Arc<StoreNew>) -> Result<
 
     // The generated OpenAPI document contains "discriminator" fields which are not generated correctly to be supported by redoc
     // We need to remove them from the document to make sure redoc can render the document correctly
-    let original_doc = serde_json::to_value(&ApiDoc::openapi())
+    let original_doc = serde_json::to_value(ApiDoc::openapi())
```

Comment:
Why we should change this line?

Reply:
not sure, the linter automatically did this.